docs/view_components/flow_chart.rst

```diff
@@ -1,11 +1,57 @@
-Component title
----------------
+.. _flowchart-label:
+
+Flow-chart
+----------
+The flow chart is a view component that shows all the sequential steps of the tool in order to get results.
+
+A flow chart view-component has a number of advantages:
+
+* Helps in sequentially progressing through the modeling and simulation process
+* Allows the user to jump between different steps of the simulation
+* Brings a design consistency and uniformity to the UI of the tool
+* Splits the tool in different views which increases the user friendliness 
 
 Attributes
 ^^^^^^^^^^
 
+* Each button is a step in the simulation process
```

Comment:
If you answer the question "what are the attributes which characterize the view-component" (not what they do or how can I interact with them) you get what should be listed under attributes. 

Here you say that a button is a step before defining steps. Besides, the fact that it should be a button is something that I would propose in the rendering part: there is a list of steps (attributes) + user can click on step (action) = steps can be button (rendering).